hack/update-api-reference-docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
How about move the hack/gen-swagger-doc/run-gen-swagger-docs.sh to hack/ and rename it? I feel there are too many layers.